src/Sulu/Bundle/AdminBundle/Resources/js/components/Block/Block.js

```diff
@@ -49,13 +70,13 @@ export default class Block<T: string> extends React.Component<Props<T>> {
 
     render() {
         const {
+            actions,
             activeType,
             children,
             dragHandle,
             icons,
             onCollapse,
             onExpand,
-            onRemove,
```

Comment:
TODO: describe how to upgrade a custom component that used this property in the `UPGRADE.md`

Reply:
Should we just migrate it to actions if it is defined to avoid a bc break here. Do we know if somebody did use the Block component standalone yet? /cc @sulu/core-team